From be740c27dd974facf13acb469be48fa7025f7c29 Mon Sep 17 00:00:00 2001
From: Julien Gilli <julien.gilli@joyent.com>
Date: Wed, 6 Sep 2017 11:28:10 -0400
Subject: [PATCH] ZAPI-798 moray bucket initialization integration test fails
 due to recent moray changes Reviewed by: Cody Peter Mello
 <cody.mello@joyent.com> Reviewed by: Trent Mick <trent.mick@joyent.com>
 Approved by: Trent Mick <trent.mick@joyent.com>

---
 test/vms.update-moray-bucket-non-transient-error.test.js | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/test/vms.update-moray-bucket-non-transient-error.test.js b/test/vms.update-moray-bucket-non-transient-error.test.js
index 694c48a..d1c132c 100644
--- a/test/vms.update-moray-bucket-non-transient-error.test.js
+++ b/test/vms.update-moray-bucket-non-transient-error.test.js
@@ -164,7 +164,8 @@ exports.moray_init_non_transient_error = function (t) {
 
         vmapiClient.ping(function onVmapiPing(pingErr, obj, req, res) {
             var errBody = pingErr.body;
-            var expectedErrString = 'docker.type is invalid';
+            var expectedErrString = 'bucket.index[\'docker\'].type should be ' +
+                'equal to one of the allowed values';
             var expectedHealthiness = false;
             var expectedResponseHttpStatus = 503;
             var expectedStatus = 'some services are not ready';
-- 
2.21.0

